web/src/markdown.ts

```diff
@@ -200,13 +200,13 @@ function parse_with_options(
                 let classes;
                 let display_text;
                 if (silently) {
-                    classes = "user-mention silent";
+                    classes = "user-mention user-mention-all silent";
                     display_text = mention;
                 } else {
                     // Stream Wildcard mention
                     mentioned_stream_wildcard = true;
                     display_text = "@" + mention;
-                    classes = "user-mention";
+                    classes = "user-mention user-mention-all";
```

Comment:
I think `mention-channel` would be a better name for this than `user-mention-all`; we use `topic-mention` for the parallel class when mentioning a topic.